src/components/ui/home/index.tsx

```diff
@@ -50,7 +50,7 @@ export const Intro = () => {
         </h1>
 
         <p className={'text-pretty flex flex-col gap-2'}>
-          <span style={{ maxWidth: '42ch' }}>
+          <span className={'md:max-w-[26rem]'}>
```

Comment:
Please double check these values, they are meant to leave a "safe" empty space around the photo.

Additionally, let's not use a breakpoint, `max-w` will be safe in mobile devices
```suggestion
          <span className={'max-w-[26rem]'}>
```

![Shot 2024-03-21 at 16 04 40@2x](https://github.com/jahirfiquitiva/jahir.dev/assets/10360816/e30b010f-dc1a-4395-8787-c4dea3931e43)

Reply:
Oh, and please try using a value from the spacing configured in tailwind, rather than an arbitrary value